controllers/array_action/array_mediator_svc.py

```diff
@@ -55,6 +55,7 @@
 HOSTS_LIST_ERR_MSG_MAX_LENGTH = 300
```

Comment:
do we need a unit test for it?

Reply:
I added two unit tests - success for EAR volume with 2 fc mappings and failure for EAR volume with another number.